### PR TITLE
Add stamp directives by default to new files

### LINKED ISSUE #253: New files get default directives
We currently don't have Stamp and PBASIC directive quick-access buttons - we need to ease the pain of users creating new files.

For a temporary time, we want all new files to automatically have the default Stamp and PBASIC directives inserted into them.

```
'{$STAMP BS2}
'{$PBASIC 2.5}


```

### PR DESCRIPTION
**Pull request by urbantumbleweed**
#### What's this PR do?

It adds directives to new files and moves the cursor after the directives string.
#### What are the important parts of the code?

Look at handlers and a new file `lib/directives.js'
#### How should this be tested by the reviewer?

download the branch, build, launch and create new files in editor.
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

closes #253
#### Screenshots (if appropriate)

screen shot 2015-09-11 at 12 39 38 pm

### CODE REVIEW

File: src/lib/directives.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+function device(type){
+  return `'{$STAMP ${type}}`;
+}
+
+function pbasic(ver){
+  return `'{$PBASIC ${ver}}`;
+}
+
+const directives = {
```
Comment (phated) [inline src/lib/directives.js]: no point in exporting an object here, just export a method that takes a board and language and returns your directive string.  You are doing interpolation inside `src/plugins/handlers.js` so this module serves no purpose unless it moves that logic into a function instead.

File: src/plugins/handlers.js [hunk 2]
```diff
@@ -99,11 +100,15 @@ function handlers(app, opts, done){
       return;
     }
 
+    const directive =
+    `${boards.bs2}\r${languages.pbasic2_5}\r\r`;
```
Comment (phated) [inline src/plugins/handlers.js]: see other comment, this should be a method call, not string interpolation

File: src/plugins/handlers.js [hunk 3]
```diff
@@ -260,6 +265,10 @@ function handlers(app, opts, done){
     cm.execCommand('replace');
   }
 
+  function goToEnd(){
+    cm.execCommand('goDocEnd');
```
Comment (phated) [inline src/plugins/handlers.js]: I think this should be built into the `documents.create` method.  I don't think there is ever a scenario where you would want to create the document but not focus the end

File: src/plugins/handlers.js [hunk 4]
```diff
@@ -99,11 +100,15 @@ function handlers(app, opts, done){
...
+    const directive =
+    `${boards.bs2}\r${languages.pbasic2_5}\r\r`;
+
     workspace.newFile(builtName, '')
       .then(() => userConfig.set('last-file', builtName))
       .then(function(){
-        documents.create(path.join(cwd, builtName), '');
+        documents.create(path.join(cwd, builtName), directive);
         documents.focus();
+        goToEnd();
```
Comment (phated) [inline src/plugins/handlers.js:113]: see other comment, remove this, build into `documents.create`

**Comment by urbantumbleweed** [comment]
This is ready for review.

**Comment by urbantumbleweed** [comment]
This has been replaced by pull request #266 because of problems with this branch.

**Pull request closed without merging**